react/core/main.tsx

```diff
@@ -63,6 +65,14 @@ import {
 import withHMR from '../utils/withHMR'
 import { generateExtensions } from '../utils/blocks'
 import { promised } from '../utils/promise'
+import {
+  RenderedSuccess,
+  NamedServerRendered,
+  Rendered,
+  ServerRendered,
+} from '../typings/global'
+import { RenderRuntime, Extensions } from '../typings/runtime'
+import '../typings/runtime'
```

Comment:
Lose import?

Reply:
I need this otherwise builder hub doesn't add this file to the final bundle making types unavailable. I'll add a comment on that.